### DISCUSSION TITLE
Overriding multiple actions from one git repo

### DISCUSSION THREAD

**Discussion by DoctorRadar** [todotxt/todo.txt-cli#338] [Q&A] [answered]
A little while ago I wrote an override for the `add` action so it's always `-t add` and all entries in my todo.txt have the created on date. I thought it was cool and I shared it by making by `.todo.actions.d/add` directory a local git repo and pushing it to the github remote. Now, another user has said, "Thanks, I think it's cool too, but it doesn't work with `addm`." I can fix that by writing an override for `addm` as `-t addm` however, that new override needs to be in `.todo.actions.d/addm` which means unless I make my `.todo.actions.d` directory a git repo, which would be bad, it's hard to see how to get both `add` and `addm` overrides into a single github repository.

Anyone have any thoughts?

**Comment by inkarkat** [comment, marked as answer]
I would just create a symlink from `.todo.actions.d/add` to `.todo.actions.d/addm`.

Or maybe checkout a separate _working tree_:

    $ git worktree add -f ../addm
